### DISCUSSION TITLE
How can i use the onChange event? please answer me!!!

### DISCUSSION THREAD

**Discussion by EduardoMtzUta** [themesberg/flowbite-react#1243] [General]
How can i use the onChange event on DatePicker? please answer me!!!

**Comment by lydiapostk** [comment]
On "flowbite-react": "^0.11.7":

The expected type for onChange event is `(value: null | undefined) => void`

For my own code, I would create control variables:
`const [date, setDate] = useState<Date>(new Date())`

Set onchange:
`<Datepicker value={date} onChange={setDate} theme={yellowTheme} showClearButton={false} showTodayButton={false} />`

The type of `date` is `null | Date`.
